Source/JavaScriptCore/runtime/JSGlobalObject.cpp

```diff
@@ -677,8 +679,13 @@ JSC_DEFINE_HOST_FUNCTION(enqueueJob, (JSGlobalObject* globalObject, CallFrame* c
     JSValue argument1 = callFrame->argument(2);
     JSValue argument2 = callFrame->argument(3);
     JSValue argument3 = callFrame->argument(4);
+#if USE(BUN_JSC_ADDITIONS)
+    JSValue argument4 = callFrame->argument(5);
 
+    globalObject->queueMicrotask(job, argument0, argument1, argument2, argument3, argument4);
```

Comment:
Support passing a 4th extra argument to `queueMicrotask` to avoid jamming everything into the 3rd (required duck typing and extra checks to manage).